fixtures/category_fixture.json

```diff
@@ -0,0 +1,29 @@
+[
```

Comment:
Rename Categories too, make them real, together with descriptions. Use Chat GPT if you are too lazy to write text descriptions
;)

Reply:
I hope now its enough